api.planx.uk/editor/moveFlow.ts

```diff
@@ -0,0 +1,82 @@
+import { Request, Response, NextFunction } from "express";
+import { gql } from 'graphql-request';
+import { adminGraphQLClient as client } from "../hasura";
+import { Flow, Team } from "../types";
+
+const moveFlow = async (
+  req: Request,
+  res: Response,
+  next: NextFunction
+): Promise<Response | NextFunction | void> => {
+  try {
+    if (!req.user?.sub) {
+      return next({ status: 401, message: "User ID missing from JWT" });
+    }
+
+    if (!req.params?.flowId || !req.params?.teamSlug) {
+      return next({ status: 400, message: "Missing required values to proceed" });
+    }
+
+    // Translate teamSlug to teamId
+    const teamId = await getTeamIdBySlug(req.params.teamSlug);
+    
+    // If we have a valid teamId, update the flow record
+    if (teamId) {
+      await updateFlow(req.params.flowId, teamId);
+      res.status(200).send({
+        message: `Successfully moved flow to ${req.params.teamSlug}`,
+      });
+    } else {
+      return next({
+        status: 400,
+        message: `Unable to find a team matching slug ${req.params.teamSlug}, exiting move`,
+      });
+    }
+  } catch (error) {
+    return next(error);
+  }
+};
+
+const getTeamIdBySlug = async (slug: Team["slug"]): Promise<Team["id"]> => {
```

Comment:
I appreciate this way of doing it - `Team["id"]` and `Team["slug"]` is more useful and meaningful I think.